components/TestSubmissionsResults/TestSubmissionsResults.tsx

```diff
@@ -2,19 +2,24 @@
 import textStyles from './TestSubmissionsResults.module.scss';
 
 interface TestSubmissionsResultsProps {
-    test: string;
+    testId: string;
+    testResult: string;
+    testAnswer: string;
 }
 
+
 export function TestSubmissionsResults(props: TestSubmissionsResultsProps): JSX.Element {
-   
-    //TODO: how will results appear to admin? for now put textarea as placeholder
 
+    //TODO: how will results appear to admin? for now put textarea as placeholder
     return (
         <li>
-            <h2 className={textStyles.testTitle}>{props.test}</h2>
-            <textarea className={textStyles.textArea}>
-                Placeholder for Test Results
+            <h2 className={textStyles.testTitle}>Test {props.testId}</h2>
+            <textarea
+                className={textStyles.textArea}
+                value= {props.testAnswer}
+                readOnly={true}>
             </textarea>
+            <p className={textStyles.testTitle}>{props.testResult}</p>
         </li>
```

Comment:
Bea is about to merge in a ticket to handle this for us! :)
She will actually probably beat you to merging - so you'll probably just need to handle the merge conflict here and then it should all work! :)

Reply:
this is the PR - https://github.com/techswitch-learners/testswitch-frontend-admin/pull/20#pullrequestreview-404092107

Reply:
fun merging times ahead! 😂 Will do.

Reply:
✔️